### ISSUE TITLE
log_response is an invalid option for multiscrape

### ISSUE THREAD

**Issue by Madelena** [danieldotnl/ha-multiscrape#106]
Version of the custom_component: 5.7.0 (Installed via HACS)

Version of Home Assistant Core: core-2022.4.1
Installation Type	Home Assistant OS
Development	false
Supervisor	true
Docker	true
Virtual Environment	false
Python Version	3.9.9
Operating System Family	Linux
Operating System Version	5.10.103-v8
CPU Architecture	aarch64

## Configuration

```yaml
multiscrape:
  - resource_template: >
      {{ "https://flightaware.com/live/flight/" + states('sensor.opensky_flight_in_vicinity') }}
    name: OpenSky FlightAware Tracker
    log_response: true
    scan_interval: 360000
    sensor:
      - unique_id: opensky_flight_friendly_name
        name: Flight Friendly Name
        select: ".flightPageSummaryContainer > .flightPageFlightIdentifier > .flightPageSummary > .flightPageFriendlyIdent > .flightPageFriendlyIdentLbl > h1"
        icon: mdi:airplane
      - unique_id: opensky_origin_airport_code
        name: Origin Airport Code
        select: "#flightPageTourStep1 > div.flightPageSummaryAirports > div.flightPageSummaryOrigin > span.flightPageSummaryAirportCode > span"
        icon: mdi:airport
      - unique_id: opensky_origin_city
        name: Origin City
        icon: mdi:city
        select: "#flightPageTourStep1 > div.flightPageSummaryAirports > div.flightPageSummaryOrigin > span.flightPageSummaryCity"
      - unique_id: opensky_destination_airport_code
        name: Destination Airport Code
        select: ".flightPageSummaryAirports > .flightPageSummaryDestination > .flightPageSummaryAirportCode > span"
        icon: mdi:airport
      - unique_id: opensky_destination_city
        name: Destination City
        select: ".flightPageSummaryAirports > .flightPageSummaryDestination > .flightPageSummaryCity > span"
        icon: mdi:city
```

## Describe the bug

When I check my configuration before restart, Home Assistant warned me that adding the line "log_response: true" in my configuration.yaml is invalid.

## Debug log

```
The system cannot restart because the configuration is not valid: Invalid config for [multiscrape]: [log_response] is an invalid option for [multiscrape]. Check: multiscrape->multiscrape->0->log_response. (See /config/configuration.yaml, line 569).
```

**Comment by danieldotnl** [comment]
It's only available in the pre-release (which can be enabled in HACS).

**Comment by danieldotnl** [comment]
Now available in release v6.2.0

**Comment by Madelena** [comment]
Thanks a lot for helping! Great component!
